trio/_channel.py

```diff
@@ -141,6 +147,9 @@ def send_nowait(self, value):
 
     @enable_ki_protection
     async def send(self, value):
+        """See `~trio.abc.SendChannel.send`.
```

Comment:
This will format as "See send()", won't it? I might remove the tilde so it looks less recursive.